pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -31,6 +31,10 @@ let
       mv System/* .
       rmdir System
 
+      pushd lib
+      ln -s -L /usr/lib/libcups*.dylib .
```

Comment:
Could we maybe use the open source cups library? Apple also publishes cups sources as well but it may be closed enough to upstream cups.